blazor/datagrid/column-validation.md

```diff
@@ -399,4 +399,119 @@ Now the HandleValidation method of the MyCustomValidator component would be chan
         }).ToList();
     }
 }
-```
\ No newline at end of file
+```
```

Comment:
link this topic in Dialog editing template part to in see also or notes section.